cmake/configure_wcoss_cray.cmake

```diff
@@ -1,34 +0,0 @@
-message("")
```

Comment:
I am curious what wcoss platforms configuration files are removed, aren't they level-1 platforms?

Reply:
No, they are not supported at all for UFS MRW v1.0 or v1.1: https://github.com/ufs-community/ufs/wiki/Supported-Platforms-and-Compilers